lib/bob/job/otp_checker.ex

```diff
@@ -1,12 +1,14 @@
 defmodule Bob.Job.OTPChecker do
   @repo "erlang/otp"
   @linuxes ["ubuntu-14.04", "ubuntu-16.04", "ubuntu-18.04", "ubuntu-20.04", "ubuntu-22.04"]
+  @arches ["amd64", "arm64"]
 
   def run(_type) do
     for linux <- @linuxes,
+        arch <- @arches,
         {ref_name, ref} <- Bob.GitHub.diff(@repo, "builds/otp/#{linux}"),
         build_ref?(linux, ref_name),
-        do: Bob.Queue.add(Bob.Job.BuildOTP, [ref_name, ref, linux])
+        do: Bob.Queue.add(Bob.Job.BuildOTP, [ref_name, ref, linux, arch])
```

Comment:
We need to verify what OTP versions we can build on, I am guessing older versions may not build on arm64, but not sure.

Reply:
The arch needs to be added like this so we can fetch the jobs from machines with different architectures.

```suggestion
        do: Bob.Queue.add({Bob.Job.BuildOTP, arch}, [ref_name, ref, linux])
```